Misc/NEWS.d/next/Core and Builtins/2022-03-30-02-36-25.bpo-46775.e3Oxqf.rst

```diff
@@ -0,0 +1,4 @@
+Update the constructor of :exc:`OSError` to use winerror_to_errno()
+unconditionally. Some :exc:`OSError` now be raised as a subclass and the
+:attr:`OSError.errno` follows the mapping policy of winerror_to_errno().
+Patch by Dong-hee Na.
```

Comment:
winerror_to_errno() is an internal function, but don't mention it but explains the change to end users. What I understand is that a Windows error is now correctly converted to an ``errno`` number for error numbers greater than or equal to 10000. For example, the socket error WSAEACCES is now converted to EACCES.

Maybe even give the list of errors code with their numbers:

```
     case WSAEINTR: 
     case WSAEBADF: 
     case WSAEACCES: 
     case WSAEFAULT: 
     case WSAEINVAL: 
     case WSAEMFILE: 
```

https://docs.microsoft.com/en-us/windows/win32/debug/system-error-codes--9000-11999-

Reply:
Thanks Victor :)